packages/animations/lib/src/fade_through_transition.dart

```diff
@@ -239,7 +239,7 @@ class _ZoomedFadeInFadeOut extends StatelessWidget {
         Widget? child,
       ) {
         return _FadeOut(
-          child: child,
+          child: child!,
```

Comment:
Wouldn't that be visually pretty surprising? You expect a fade out but get blank? Although I guess that's more compatible with current behavior.

I'm curious why we didn't end up having the same problem with `_ZoomedFadeIn`.

Reply:
If we make FadeThroughTransition child required it will fix all private _ZoomedFadeInFadeOut class problems as well will help with future nullable issues with Transform widget

Reply:
> Wouldn't that be visually pretty surprising? You expect a fade out but get blank? Although I guess that's more compatible with current behavior.

Ahh yes. Something like this maybe then `child ?? const SizedBox()`.
Anyway crash is not what we want =)

> I'm curious why we didn't end up having the same problem with _ZoomedFadeIn.

That's because FadeTransition actually gets a child wrapped into ScaleTransition